great_expectations/datasource/fluent/databricks_sql_datasource.py

```diff
@@ -86,7 +86,6 @@ class _UrlSchemaError(pydantic.UrlError):
 class DatabricksDsn(AnyUrl):
     allowed_schemes = {
         "databricks",
-        "databricks+connector",
```

Comment:
Is there any reason that you can think of that someone might want to keep using the deprecated connector?
We could keep allowing it but raise a warning if it gets used.

I'm also fine with just removing it. We can always add it back if users request it.